department-of-veterans-affairs/va.gov-cms
# VACMS-9358: add audit opening hours comments field script

## Description

Closes #9358 

## Testing done

Visual / Manual testing

## Screenshots

![Screen Shot 2022-06-08 at 6 26 39 PM](https://user-images.githubusercontent.com/21045418/172727989-f487b837-1d70-438a-b382-7946716b863d.png)

## QA steps

From a terminal:
1. Run the command `drush scr ./scripts/content/VACMS-8935-scrub-tz-in-hours-field-comment.php`
2. Check the docroot for the project - verify a new file was created: `docroot/office_hours_comment_audit.csv`

### Select Team for PR review
